korman/ui/modifiers/gui.py

```diff
@@ -88,6 +88,9 @@ def row_alert(prop_name, **kwargs):
 
         layout.separator()
         layout.prop(modifier, "link_type")
+        if modifier.link_type == "kOriginalBook":
+            layout.prop(modifier, "shareable")
+            layout.prop(modifier, "share_region")
```

Comment:
Seeing as how this only works on MOUL, I wonder if it should be moved down into the `if "pvMoul" in modifier.versions` section below. That way, it doesn't appear in TPotS only linking books.

Reply:
So you're thinking further down in the other `if "pvMoul"` among `link_destination` and `spawn_title` (presumably before or after both of those)?

Reply:
Or you could change the `if` statement here to `if "pvMoul" in modifier.versions and modifier.link_type == "kOriginalBook"`. Whichever one makes the most sense to you in the UI.

Reply:
OK, think I got things looking the way I want. Thoughts are welcome.